library/src/main/java/library/service/BookServiceImpl.java

```diff
@@ -0,0 +1,168 @@
+package library.service;
+
+import library.XML.ParserDOM;
+import library.XML.ValidatorXML;
+import library.dao.BookDao;
+import library.dao.BookDaoImpl;
+import library.model.Author;
+import library.model.Book;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class BookServiceImpl implements BookService {
+
+    BookDao bookDao = new BookDaoImpl();
+    Scanner scanner = new Scanner(System.in);
+
+    @Override
+    public void createBook() {
+        try {
+            Book book = new Book();
+            System.out.println("Введите название книги:");
+            book.setTitle(scanner.nextLine());
+            System.out.println("Введите имя автора:");
+            book.setAuthor(new Author(scanner.nextLine()));
+            System.out.println("Введите ISBN:");
+            book.setISBN(scanner.nextLine());
+            book.setGenre(new GenreServiceImpl().genreChoice());
```

Comment:
not good to call one part of logic while another is in progress. I mean book creation depends on genre but it shouldn't

Reply:
and every time you want to choose genre you creating new object of GenreServiceImpl